app/src/screens/VerificationSteps.tsx

```diff
@@ -0,0 +1,134 @@
+import { Link, useTheme } from '@hyperledger/aries-bifold-core'
+import { useNavigation } from '@react-navigation/native'
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Text, View, StyleSheet, FlatList, Pressable } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+import { BCScreens } from '../navigators/navigators'
+
+interface SetupStep {
+  title: string
+  active: boolean
+  complete: boolean
+  content: JSX.Element
+  onPress?: () => void
+}
+
+const VerificationSteps: React.FC = () => {
+  const { ColorPallet, TextTheme } = useTheme()
+  const navigation = useNavigation()
+  const { t } = useTranslation()
+
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      backgroundColor: ColorPallet.brand.primaryBackground,
+    },
+    itemSeparator: {
+      width: '100%',
+      height: 8,
+      backgroundColor: ColorPallet.brand.primaryBackground,
+    },
+    step: {
+      paddingVertical: 24,
+      paddingHorizontal: 24,
+      flex: 1,
+    },
+    contentContainer: {
+      marginTop: 16,
+      flex: 1,
+    },
+    titleRow: {
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
+    contentText: {
+      flex: 1,
+      flexWrap: 'wrap',
+      fontSize: 16,
+    },
+    contentEmailContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+    },
+    contentEmail: {
+      flex: 1,
+      flexWrap: 'wrap',
+      fontSize: 16,
+    },
+    contentEmailButton: {
+      alignSelf: 'flex-end',
+    },
+  })
+
+  const steps: SetupStep[] = useMemo(() => {
+    return [
```

Comment:
Just noting these might / will probably come from the server.

Reply:
True